docs/run.rst

```diff
@@ -118,28 +120,176 @@ within the source repository at ``util/scripts/lsfcsm``.
+.. code-block:: Bash
+
+    $ [prompt] cat example_stage_in.manifest
+
+    /scratch/users/me/input_data/input_1.dat /unifyfs/input/input_1.dat
+    /home/users/me/configuration/run_12345.conf /unifyfs/config/run_12345.conf
+    "/home/users/me/file with space.dat" "/unifyfs/file with space.dat"
+
+Transfer During Job
+*******************
+
+Data can also be transferred in/out of UnifyFS using one of two stand-alone
+applications.
+
+The stand-alone applications can be invoked at any time while the UnifyFS
+servers are up and responding to requests. This allows for bringing in new input
+and/or transferring results out to be verified before the job terminates.
+
+Unifyfs Stage Executable
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+``$UNFYFS_INSTALL/libexec/unifyfs-stage``
+
+The ``start``/``terminate`` transfer API stage functionality can also be used
+via the stand-alone application ``unifyfs-stage``.
+
+This application can be run at any time within a job to transfer new data into
+ or results out of UnifyFS.
+A manifest file (see :ref:`above <manifest_file_label>`_) needs to be provided
+as an argument to use this approach.
+
+.. code-block:: Bash
+
+    [prompt]$ ./unifyfs-stage --help
+
+    Usage: unifyfs-stage [OPTION]... <manifest file>
+
+    Transfer files between unifyfs volume and external file system.
+    The <manifest file> should contain list of files to be transferred,
+    and each line should be formatted as
+
+      /source/file/path /destination/file/path
+
+    OR in the case of filenames with spaces or special characters:
+
+      "/source/file/path" "/destination/file/path"
+
+    One file per line; Specifying directories is not supported.
+
+    Available options:
+      -c, --checksum           verify md5 checksum for each transfer
+      -h, --help               print this usage
+      -m, --mountpoint=<mnt>   use <mnt> as unifyfs mountpoint
+                               (default: /unifyfs)
+      -p, --parallel           transfer each file in parallel
+                               (experimental)
+      -s, --share-dir=<path>   directory path for creating status file
+      -v, --verbose            print noisy outputs
+
+    Without the '-p, --parallel' option, a file is transferred by a single
+    process. If the '-p, --parallel' option is specified, each file will be
+    divided by multiple processes and transferred in parallel.
+
+Examples:
+
+.. code-block:: Bash
+    :caption: Serial Transfer
+
+    $ srun -N 1 -n 1 $UNIFYFS_INSTALL/libexec/unifyfs-stage $MY_MANIFEST_FILE
+
+.. code-block:: Bash
+    :caption: Parallel Transfer
+
+    $ srun -N 4 -n 8 $UNIFYFS_INSTALL/libexec/unifyfs-stage --parallel $MY_MANIFEST_FILE
+
+Transfer Executable
+^^^^^^^^^^^^^^^^^^^
+
+``$UNFYFS_INSTALL/libexec/transfer-static``
+
+.. note::
+
+    The ``transfer-gotcha`` executable is currently unusable due to an issue
+    that is being tracked.
+
+The manifest subsystem can also be used during the job by invoking the
```

Comment:
I don't think "manifest subsystem" applies in this case. Maybe substitute with "transfer API"?